docs/advanced/security.md

```diff
@@ -0,0 +1,254 @@
+      containers:
+      - name: edgemesh-server
+        image: kubeedge/edgemesh-server:test
+        imagePullPolicy: IfNotPresent
+        env:
+          - name: MY_NODE_NAME
+            valueFrom:
+              fieldRef:
+                fieldPath: spec.nodeName
+        ports:
+        - containerPort: 10004
+          name: relay
+          protocol: TCP
+        resources:
+          limits:
+            cpu: 200m
+            memory: 1Gi
+          requests:
+            cpu: 100m
+            memory: 512Mi
+        volumeMounts:
+          - name: conf
+            mountPath: /etc/kubeedge/config
+          - name: edgemesh
+            mountPath: /etc/kubeedge/edgemesh
+          # mount token information  
+          - name: ca-server-token
+            mountPath: /etc/kubeedge/cert
+      restartPolicy: Always
+      serviceAccountName: edgemesh-server
+      volumes:
+        - name: conf
+          configMap:
+            name: edgemesh-server-cfg
+        - name: edgemesh
+          hostPath:
+            path: /etc/kubeedge/edgemesh
+            type: DirectoryOrCreate
+        # add token information
+        - name: ca-server-token
+          secret:
+            secretName: tokensecret
+```
+3. redeploy
+Once the changes above have been made, we can redeploy edgemesh-server directly
+
+### edgemesh-agent configuration
+1. config edgemesh-agent's configmap
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: edgemesh-agent-edge-cfg
+  namespace: kubeedge
+  labels:
+    k8s-app: kubeedge
+    kubeedge: edgemesh-agent
+data:
+  edgemesh-agent.yaml: |
+    apiVersion: agent.edgemesh.config.kubeedge.io/v1alpha1
+    kind: EdgeMeshAgent
+    commonConfig:
+      dummyDeviceName: edgemesh0
+      dummyDeviceIP: 169.254.96.16
+    kubeAPIConfig:
+      burst: 200
+      contentType: application/json
+      kubeConfig: ""
+      master: "http://127.0.0.1:10550"
+      qps: 100
+    goChassisConfig:
+      protocol:
+        tcpBufferSize: 8192
+        tcpClientTimeout: 5
+        tcpReconnectTimes: 3
+      loadBalancer:
+        defaultLBStrategy: RoundRobin
+        supportLBStrategies:
+          - RoundRobin
+          - Random
+          - ConsistentHash
+        consistentHash:
+          partitionCount: 100
+          replicationFactor: 10
+          load: 1.25
+    modules:
+      edgeDNS:
+        enable: true
+        listenInterface: docker0
+        listenPort: 53
+      edgeProxy:
+        enable: true
+        subNet: 10.96.0.0/12
+        listenInterface: docker0
+        listenPort: 40001
+      edgeGateway:
+        enable: false
+        nic: "*"
+        includeIP: "*"
+        excludeIP: "*"
+      tunnel:
+        enable: true
+        listenPort: 10006
+        publicIP: "<public-ip>"
+        # 插入如下内容
+        enableSecurity: true
+        tunnelACL:
+          # cloudcore's https api for crt sign apply
+          httpServer: <cloudhub-https-addr>
+```
+2. config edgemesh-agent's deployment
```

Comment:
same question as server: can be unified as one deployment regardless of security connect or not